angular/src/lib/radio/radio.component.ts

```diff
@@ -44,6 +44,10 @@ const CUSTOM_RADIO_VALUE_ACCESSOR: any = {
       <span>{{ label }}</span>
     </label>
 
+    <!-- Helper Text -->
+    <md-input-helper *ngIf="helpText" [message]="helpText">
+    </md-input-helper>
```

Comment:
Should we use something else here?  or add a class so that the help text moves over and isn't right under the radio? @pauljeter @bfbiggs

Reply:
No, the CSS is currently broken. We need to fix in core for all of the libraries.